src/smc-webapp/editor_chat.cjsx

```diff
@@ -699,11 +702,24 @@ ChatRoom = (name) -> rclass
             e.preventDefault()
             @clear_input()
         else if e.keyCode==13 and e.shiftKey # 13: enter key
-            @send_chat(e)
+            @setState(realtime_typing_message_id : '')
+            if not @state['realtime_typing_message_id']
```

Comment:
State references by fixed strings should use dot notation ie. `@state.realtime_typing_message_id`.